File: Java/EuclideanAlgo.java
// Euclidean Algorithm is used to find GCD(Greatest Commin Divisor) of two numbers
import java.util.*;
public class EuclideanAlgo {
	public static void main(String args[]) {
		int a = 35, b = 10;
		System.out.println(gcd(a, b));

	}

	private static int gcd(int a, int b) {
		if(b <= 0)
			return a;

		return gcd(b, a%b);
	}

}

File: Python/EuclideanAlgo.py
# Python program to find gcd of two numbers  
  
# Function to return gcd of a and b 
def gcd(a, b):  
    if a == 0 : 
        return b  
      
    return gcd(b%a, a) 
  
a = 10
b = 15
print("gcd(", a , "," , b, ") = ", gcd(a, b)) 
